src/ui/Select/Select.jsx

```diff
@@ -197,6 +204,15 @@ const Select = forwardRef(
             {isOpen && (
               <>
                 {items.map(_renderItem)}
+                {variant === 'defaultOrgSelector' &&
+                  providerToName(provider) === 'Github' && (
+                    <li className="px-4 py-2 text-ds-blue-darker">
+                      <A to={{ pageName: 'codecovAppInstallation' }}>
+                        <Icon name="plus-circle" />
+                        Add GitHub organization
+                      </A>
+                    </li>
+                  )}
```

Comment:
This section here needs to be lifted out of the `ui/Select` anything in the `ui` directory cannot have this level of nuance to it as they are the building blocks. In this instance you will have to append this item to your list of orgs where you are using the `Select`.